src/main/java/gov/cabinetofice/gapuserservice/model/User.java

```diff
@@ -38,6 +41,14 @@ public class User {
     @Enumerated(EnumType.STRING)
     private LoginJourneyState loginJourneyState;
 
+    @CreatedDate
+    @Column(name = "created")
+    private Instant created;
```

Comment:
Will this column populate when the user row is created? Do we instead want to set this value when the user runs through OL `/login`

Reply:
Yes, I spoke to Dylan about this one: 
<img width="1120" alt="Screenshot 2023-08-18 at 15 04 59" src="https://github.com/cabinetoffice/gap-user-service/assets/107405237/5b4c81a3-4f58-4c02-a1f1-d45a76816899">

Reply:
I believe @CreatedDate will automatically only create the date when a user resource is added.

I can see that we are saving the user again after the user logs in via One login

` public User createOrGetUserFromInfo(final OneLoginUserInfoDto userInfo) {
        final Optional<User> userOptional = getUserFromSubOrEmail(userInfo.getSub(), userInfo.getEmailAddress());
        if (userOptional.isPresent()) {
            final User user = userOptional.get();
            if (!user.hasSub()) {
                user.setSub(userInfo.getSub());
                return userRepository.save(user);
            }
            return user;
        }
        return createNewUser(userInfo.getSub(), userInfo.getEmailAddress());
    }`
    
    
    Can you confirm this is adds the dates automatically @kiramarstonTCO ?